pkg/scheduler/framework/cyclestate.go

```diff
@@ -32,6 +37,16 @@ type CycleStatePluginReadWriter interface {
 type CycleState struct {
 	// store is a concurrency-safe store (a map).
 	store sync.Map
+
+	// skippedFilterPlugins is a set of Filter plugins that should be skipped in the current scheduling cycle.
+	skippedFilterPlugins sets.String
```

Comment:
Hi Ryan! The reason why generic set is not used is that it is not supported yet with our current dependency version. It was added since 1.26; ours use 1.25. Is it OK if I bump version?